src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -362,9 +338,20 @@ var ReactCompositeComponentMixin = {
       markup = this.performInitialMount(renderedElement, hostParent, hostContainerInfo, transaction, context);
     }
 
+    var debugID = 0;
```

Comment:
It’s not set in production, and accessing non-existent field is a potential perf issue because it traverses the prototypes.